drafts/prometheus-alertmanager.adoc

```diff
@@ -0,0 +1,250 @@
+
+image::/img/blog/prometheusAM_ui_status.png[Prometheus Status on UI, align="left"]
+
+You're now ready to define which metrics you want Prometheus to collect.
+
+=== Creating Prometheus alerting rules
+
+Create some alerting rules in Prometheus to define which metrics will be received by Prometheus for passing on to Slack.
+
+. Make a file called `alert.yml`. (JENNIFER: Where?) Inside that file, create some alerting rules, for example:
++
+```
+groups:
+- name: libertyexample
+  rules:
+  - alert: heapUsageTooHigh
+    expr: base_memory_usedHeap_bytes / base_memory_maxHeap_bytes > 0.9
+    for: 1m
+    labels:
+      severity: warning
+    annotations:
+      summary: "Heap usage is too high"
+      description: "{{ $labels.instance }} heap usage is too high"
+```
++
+This rule will send an alert called `heapUsageTooHigh` when the PromQL query `base_memory_usedHeap_bytes / base_memory_maxHeap_bytes` is greater than `0.9` (90%) for one minute. (For testing purposes, feel free to change the threshold to something smaller, like `0.05`.)
++
+If you're new to PromQL, check out link:https://prometheus.io/docs/prometheus/latest/querying/basics/[Querying Prometheus].
++
+. In the `prometheus.yml` file, add the location of your `alert.yml` file under `rule_files`. If it's not in the same directory as Prometheus, make sure you supply the full path.
+. Restart Prometheus. Navigate to `http://localhost:9090` and click on `Alerts` on the navigation bar. You should be able to see your alert.
+
+image::/img/blog/prometheusAM_ui_alerts.png[Prometheus Alerts on UI, align="left"]
+
+== Configure Prometheus Alertmanager to pass alerts to Slack
+
+JENNIFER:  Add a brief introduction here to say what the reader will achieve/learn from this section (configure Prometheus Alertmanager to work with Slack and then  xxxx). 
+
+
+=== Configuring the Prometheus Alertmanager to communicate with Slack
+
+Now that Prometheus is set up with rules for our Liberty server, we can set up the Prometheus Alertmanager to connect to Slack.
+
+. Edit `alertmanager.yml` to add your Slack webhook under `global`.
++
+```
+  global:
+    slack_api_url: <your api url>
+```
++
+. Under the `route`, you will see that there is a receiver called `web.hook`. We will be editing this receiver to work with Slack. Right under `webhook_configs`, add another field (aligned with `webhook_configs`) called `slack_configs`. It'll look something like this:
++
+```
+slack_configs:
+- channel: 'prometheus-alertmanager-test'
+  title: "{{ range .Alerts }}{{ .Annotations.summary }}\n{{ end }}"
+  text: "*Description*: {{ .CommonAnnotations.description }}\n*Severity*: {{ .CommonLabels.severity }}""
+```
++
+The `CommonAnnotations` come from your `alert.yml` file. The text is written using the link:https://golang.org/pkg/text/template/[Go templating] system.
++
+. Run `./alertmanager`.
```

Comment:
Not sure if this is a problem people will run into, but I think it doesn't hurt to have this information available anyway.

ADD: If you're running Alertmanager locally and running into an error, you may need run the command with an additional parameter: `./alertmanager --cluster.listen-address=""`.